ckanext/nhm/lib/utils.py

```diff
@@ -6,17 +6,22 @@
 
 from ckan.plugins import toolkit
 import requests
+from cachetools import cached, TTLCache
 
 
+@cached(cache=TTLCache(maxsize=1024, ttl=300))
```

Comment:
Presumably the `maxsize=1024` here is just an arbitary number given there's no arguments on the function so there will always only ever be 1 item in the cache? Doesn't need to be changed or anything, just checking my understanding is correct.

Also, unless I'm misunderstanding how cachetools implements a TTL cache, this will cache the function result for 5 mins but that 5 mins TTL will be reset each time the function is called. So if we still tie calling this function to page requests and we consistently get at least 1 page request every 5 mins, the status will never be updated as the cached copy will live forever?

Reply:
`maxsize` is probably unnecessary, I'll remove it.
The TTL doesn't get reset every time it's requested so it should be fine (confirmed by me testing and you going through the code)